get-involved.md

```diff
@@ -4,46 +4,47 @@ title: Get Involved
 order: 2
 ---
 
-Interested? You can start working with us in a few quick steps:
+Start by checking out our [Vision]({{ site.baseurl }}/about) and [Code of Conduct]({{ site.baseurl }}/code-of-conduct), then:
 
-1. Come say hi on our {% include icon-matrix.html %} chat
-  - Create an account to join `#tomesh` and introduce yourself on our `#welcome` channel!
-  - If you plan on connecting via a [mobile app](https://riot.im/), make sure you set the homeserver to `https://matrix.tomesh.net` under 'Use custom server options (advanced)'.
+1. Say hi on our chat {% include icon-matrix.html %}
+    - Create an account to join `#tomesh` and introduce yourself on our `#welcome` channel!
+    - If you connect via the [Riot mobile app](https://riot.im/), select _Use custom server options (advanced)_ and set the homeserver to `https://matrix.tomesh.net`
 
-2. Check out our [vision]({{ site.baseurl }}/about) and [Code of Conduct]({{ site.baseurl }}/code-of-conduct)
+1. Join one of our [events]({{ site.baseurl }}/events)
+    - Include in-person and virtual meetups and workshops
 
-3. Join us at one of our [events]({{ site.baseurl }}/events)!
-  - Events include in-person and virtual meetups and workshops
-
-4. If you have an idea or time to help, [work with us](#work-with-us)
-  - The current status of the project is tracked on our publicly-viewable [Kanban board](https://wekan.tomesh.net/b/LWS8X7sGFXqDgZ7ag/tomesh-net) and [GitHub account](https://github.com/tomeshnet)
+1. Review the current project status and [work with us](#work-with-us)
+    - [Publicly-viewable Kanban board](https://wekan.tomesh.net/b/LWS8X7sGFXqDgZ7ag/tomesh-net) keeps track of our many projects
```

Comment:
Hmm... help me out how to write this part. I want the link to align in front. The page looks much better with the highlighted text lined up. This page used to be so hard to read with all the different sized headers and links everywhere.